nvm.sh

```diff
@@ -1044,6 +1044,20 @@ nvm_install_node_source() {
     make='gmake'
     MAKE_CXX="CXX=c++"
   fi
+
+  if [ "_$NVM_OS" = "_linux" ]; then
```

Comment:
I'd still like to see this change made - feature-detection is better than UA-sniffing :-)